validation-test/stdlib/Collection/LazyFilterCollection.swift.gyb

```diff
@@ -124,25 +123,29 @@ CollectionTests.test("LazyFilterCollection instances (${traversal}${kind})") {
     var resiliency = CollectionMisuseResiliencyChecks.all
     resiliency.callNextOnExhaustedGenerator = false
 
-    let expected = [2, 4, 6, 8, 10, 12, 14, 16]
-    let base = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16]
+    let base = randArray(500)
+    var expected : [Int] = []
+    for element in base where element % 3 == 0 || element % 5 == 0 {
```

Comment:
What do you think about this change?

```swift
let testFilter: Int -> Bool = { $0 % 3 == 0 || $0 % 5 == 0 }
for element in base where testFilter(element) {
...
MinimalSequence(elements: base).lazy.filter(testFilter)
...
MinimalCollection(elements: base).lazy.filter(testFilter)
...
Minimal${traversal}${kind}(elements: base).lazy.filter(testFilter)
```

I _think_ that would work. It would be a little DRY-er. The name `testFilter` could probably be better though.

Thoughts?